packages/app-desktop/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+	'overrides': [
+		{
+			files: ['**/*.tsx', '**/*.js', '**/*.ts'],
+			rules: {
+				'no-restricted-globals': ['error',
+					...['alert', 'confirm', 'prompt'].map(alertLikeFunction => ({
```

Comment:
```suggestion
					...['alert', 'confirm'].map(alertLikeFunction => ({
```

`prompt` isn't supported in Electron at all (it throws `Uncaught Error: prompt() is not supported.`). It may make sense to include `prompt` in a separate eslint rule.